### PR TITLE
Boards index styles improvements

### PR DESCRIPTION
**Pull request by matteo95g**
Screen Shot 2021-03-15 at 12 51 35 (2)

### CODE REVIEW

File: app/components/tittle_component.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class TittleComponent < ViewComponent::Base
```
Comment (brunvez) [inline app/components/tittle_component.rb]: `Tittle`?

Comment (brunvez) [reply app/components/tittle_component.rb]: Also, this is something we would like to do with a utility class instead of a component

**Comment by matteo95g** [comment]
@mwolman I fixed the comments that you left me in #8. I tried to push them last Friday but github had some issues and I couldn't. Sorry for marking them as resolved!

**Review by brunvez — APPROVED** [review]

File: app/views/boards/_board.html.erb [hunk 2]
```diff
@@ -1,6 +1,17 @@
-<tr id="<%= dom_id board %>">
-  <td><%= board.name %></td>
-  <td><%= link_to 'Show', board %></td>
-  <td><%= link_to 'Edit', edit_board_path(board) %></td>
-  <td><%= link_to 'Destroy', board, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-</tr>
+<div class='flex flex-col border px-4 py-2 bg-rs-gray-100 rounded shadow xl:w-52 xl:h-28 justify-self-center' id="<%= dom_id board %>">
```
Comment (mwolman) [inline app/views/boards/_board.html.erb]: This line it's a little bit long (136)

What do you think about this change?
```suggestion
<div 
  class='flex flex-col border px-4 py-2 bg-rs-gray-100 rounded shadow xl:w-52 xl:h-28 justify-self-center' 
  id="<%= dom_id board %>"
>
```

Comment (matteo95g) [reply app/views/boards/_board.html.erb]: Done!

**Review by mwolman — APPROVED** [review]

File: app/javascript/stylesheets/base.scss [hunk 3]
```diff
@@ -1,5 +1,7 @@
 .link {
-  @apply font-medium;
-  @apply text-black;
-  @apply hover:text-gray-700;
+  @apply font-medium text-black hover:text-gray-700;
+}
+
+.title {
+  @apply text-4xl mb-4 text-center;
```
Comment (JuannFerrari) [inline app/javascript/stylesheets/base.scss]: if we are adding just a property let's make this a one-liner, like

`.title { @apply text-4xl mb-4 text-center; }`

same with` .link`

**Review by JuannFerrari — APPROVED** [review]

**Pull request merged**